static/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/information/index.tsx

```diff
@@ -65,63 +65,89 @@ function Information({
     return null;
   }
 
-  function getTimeSinceTooltipDescription(dateCreated: string) {
+  function getTimeSinceData(dateCreated: string) {
     const dateTime = <DateTime date={dateCreated} />;
 
     if (candidate.download.status === CandidateDownloadStatus.OK) {
-      return dateTime;
+      return {
+        tooltipDesc: dateTime,
+        displayIcon: false,
+      };
     }
 
     const uploadedBeforeEvent = moment(dateCreated).isBefore(eventDateReceived);
 
     if (uploadedBeforeEvent) {
       if (hasReprocessWarning) {
-        return (
+        return {
```

Comment:
Can we show this warning if and only if `displayIcon` is also true? Basically the notice and the icon should refer to the same things.